.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+**TL;DR**
+<One to two sentence description of the issue you are encountering or trying to solve.>
+
+### Extra Detail
+
+#### Screenshots
+<If bug fix, please upload images of the error to better showcase the problem.>
+
+#### Reasoning
+<If new idea or direction, please create a checklist of proposed ideas or next steps. [Example](https://github.com/github/training-kit/issues/415).>
+
+### How to Contribute
+- [ ] Comment on this issue if you'd like to work/collaborate on it!
+- [ ] Fork the [repository](https://github.com/github/training-kit).
+- [ ] Clone YOUR forked version of the repository to your command line. `git clone YOURURL`
+- [ ] Make changes.
+- In your command line:
+  - [ ] ADVANCED/OPTIONAL: If you'd like to have a copy of the files to be served from a web server inside of your firewall, start by running `script/package`.
+  - [ ] Run `script/setup`
+  - [ ] Run `script/server`.
+    - When successful, the script will initiate a local server at `http://127.0.0.1:4000/on-demand`.
+  - [ ] Simply paste that into your favorite web-browser and you will be ready to test.
+- [ ] Push your changes to your repository. `git push`
+- [ ] Submit a Pull Request
+  - Base Dropdown: THIS Training Kit
+  - Compare Dropdown: YOUR Training Kit repository
+
+### Questions?
+- [![Join the Open Source Training chat](https://githubtraining.herokuapp.com/badge.svg)](https://githubtraining.herokuapp.com/)
+- Leave a comment on this issue! Make sure to use @ mentions if you want a specific person's attention!
```

Comment:
Should we start identifying the Assignees is a great way to know who is 'actively' working on something and/or a major contributor to the project? Since this is a template, do we want to include an @ mention example, or are we afraid we will get a bunch of pings because people won't remove it.